src/js/index.js

```diff
@@ -691,11 +691,36 @@ export const hpe = deepFreeze({
     },
   },
   radioButton: {
-    color: 'text-strong',
+    border: {
+      color: 'border',
+      width: 'xsmall',
+    },
     check: {
-      color: 'text-strong',
+      color: 'selected-background',
+      extend: ({ theme }) => `
+        background-color: ${
+          theme.global.colors['background-front'][theme.dark ? 'dark' : 'light']
```

Comment:
I think it will make more sense to make the color as part of the theme.

Reply:
Agree. I've been noting [desired enhancements here](https://github.com/hpe-design/design-system/issues/518#issuecomment-639097244), have added this item, and will create issues resulting from this ticket.